plugins/modules/rma_workflow_manager.py

```diff
@@ -466,7 +468,9 @@ def get_have(self):
                 valid_identifier_found = True
 
                 # Check if faulty device exists
-                faulty_device_id, faulty_device_serial_number = self.device_exists(faulty_identifier, faulty_key)
+                (faulty_device_id, faulty_device_serial_number, faulty_device_name,
```

Comment:
How do we ensure that we got 7 return values from this API. I could see that we are returning only two values in few cases.

Reply:
The thing is, all the details are getting from one device only. So, if device present we are getting all the 7 values and if not "Replacement device not found in Cisco Catalyst Center." will be printed

Reply:
The question is how do we ensure that the API will return 7 values

Reply:
For both faulty and replacement device to ensure the 7 values are coming I gave added the error_message to check .
               
                # Check if faulty device exists
                (faulty_device_id, faulty_device_serial_number, faulty_device_name, faulty_device_family_name, faulty_device_series_name,
                 faulty_device_reachability_status, faulty_device_platform_id, error_message) = self.device_exists(faulty_identifier, faulty_key)
                if error_message:
                    self.msg = "Faulty device '{0}' not found in Cisco Catalyst Center".format(faulty_identifier)
                    self.log(self.msg, "ERROR")
                    self.status = "failed"
                    return self